test/native/integration-test-helpers/with-fake-timers.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Set up fake timers for executing a function and restores them afterwards.
+ *
+ * @param {Function} fn Function to trigger.
+ *
+ * @return {*} The result of the function call.
+ */
+export async function withFakeTimers( fn ) {
```

Comment:
Is the intention of this helper that we avoid using Jest's fake timer methods directly or is this a one-off utility for a specific purpose?

Reply:
The main goal is to simplify the use of Jest's fake timer methods, although I foresee that it will be mainly used by other helpers ([example](https://github.com/WordPress/gutenberg/blob/bce4e981ee8f055a40867bb6189f7bc5469a02df/test/native/integration-test-helpers/wait-for-store-resolvers.js#L24)). However, it would also help to prevent the issue related to `requestAnimationFrame` and `cancelAnimationFrame` global methods when restoring the real timers.

https://github.com/WordPress/gutenberg/blob/bce4e981ee8f055a40867bb6189f7bc5469a02df/test/native/integration-test-helpers/with-fake-timers.js#L11-L15

Reply:
Alright. That makes sense. I can foresee contributors continuing to utilize Jest timer APIs directly, which it seems like we are comfortable with. This new helper is in place to address common issues within this repository while using fake timers. 👍🏻